articles/search/search-region-support.md

```diff
@@ -35,105 +35,103 @@ You can create an Azure AI Search resource in any of the following Azure public
 
 ### Americas
 
-| Region | AI integration | Semantic ranker | Availability zones |
-|--|--|--|--|
-| Brazil South​​ ​ | ✅ | ✅ | |
-| Canada Central​​ | ✅ | ✅ | ✅ |
-| Canada East​​ ​ |  | ✅ | |
-| East US​ <sup>1</sup>| ✅ | ✅ | ✅ |
-| East US 2 ​ | ✅ | ✅ | ✅ |
-| ​Central US​​ <sup>2</sup> | ✅ | ✅ | ✅ |
-| North Central US​ ​ | ✅ | ✅ | |
-| South Central US​ <sup>1</sup>​ | ✅ | ✅ | ✅ |
-| West US​ ​ | ✅ | ✅ | |
-| West US 2​ ​ | ✅ | ✅ | ✅ |
-| West US 3​ <sup>1</sup>​ | ✅ | ✅ |✅ |
-| West Central US​ ​ | ✅ | ✅ | |
-
-<sup>1</sup> Currently, this region is at full capacity and not accepting new search services. 
-
-<sup>2</sup> This region has capacity, but some tiers are [not available](search-sku-tier.md#region-availability-by-tier). 
+| Region | AI integration | Semantic ranker | Availability zones | Capacity constrained |
+|--|--|--|--|--|
+| Brazil South​​ ​ | ✅ | ✅ | |  |
+| Canada Central​​ | ✅ | ✅ | ✅ |  |
+| Canada East​​ ​ |  | ✅ | |  |
+| East US​ <sup>1</sup>| ✅ | ✅ | ✅ | <span class='red-x'>&#10060;</span> |
+| East US 2 ​ | ✅ | ✅ | ✅ | |
+| ​Central US​​ <sup>1</sup> | ✅ | ✅ | ✅ |  |
+| North Central US​ ​ | ✅ | ✅ | |  | 
+| South Central US​ <sup>1</sup>​ | ✅ | ✅ | ✅ | <span class='red-x'>&#10060;</span> |
+| West US​ ​ | ✅ | ✅ | |  |
+| West US 2​ ​ | ✅ | ✅ | ✅ | |
+| West US 3​ <sup>1</sup>​ | ✅ | ✅ |✅ |  <span class='red-x'>&#10060;</span> |
+| West Central US​ ​ | ✅ | ✅ | | |
+
+<sup>1</sup> Capacity constraints apply to specific tiers. See tier availability by region for details [not available](search-sku-tier.md#region-availability-by-tier).
```

Comment:
@Angie, the last sentence still doesn't make sense -> "for details not available".   Just get rid of the "not available".  For example, See [tier availability by region](LINK) for details.

Reply:
thank you, removed it.